package.json

```diff
@@ -22,7 +22,7 @@
     "deploy": "npm run deploy:s3 && npm run deploy:fastly",
     "deploy:fastly": "node ./bin/configure-fastly.js",
     "deploy:s3": "npm run deploy:s3:all && npm run deploy:s3:svg && npm run deploy:s3:js",
-    "deploy:s3cmd": "s3cmd sync -P --delete-removed --add-header=Cache-Control:no-cache,public,max-age=3600",
+    "deploy:s3cmd": "s3cmd sync -P --delete-removed --add-header=Cache-Control:no-cache,public,max-age=3600 --add-header=x-amz-meta-surrogate-key:static_assets",
```

Comment:
(This was sitting since 5mo ago, just submitting it now since I got a notification on this PR)

Reply:
@rschamp The surrogate keys only operate within the service, right? So "scratch-www" is implicit.

Reply:
Well it is within the scratch-www service, but since it is our front door, several types of requests go through it, including scratchr2 and file uploader that we may want to distinguish between more systematically.  In my suggestion `scratch-www` refers to this repo rather than the service.